src/lightgbm/src/main/python/LightGBMRegressor.py

```diff
@@ -31,8 +32,33 @@ def saveNativeModel(self, sparkSession, filename):
         jsession = sql_ctx.sparkSession._jsparkSession
         self._java_obj.saveNativeModel(jsession, filename)
 
+    @staticmethod
+    def loadNativeModelFromFile(sparkSession, filename, labelColName="label",
+                                featuresColName="features", predictionColName="prediction"):
+        """
+        Load the model from a native LightGBM text file.
+        """
+        jsession = sparkSession._jsparkSession
```

Comment:
likewise there is a python equivalent to SparkSession.getOrCreate. Its something like getActiveSession

Reply:
done!